rdkit/Chem/Fingerprints/FingerprintMols.py

```diff
@@ -321,19 +320,21 @@ def _clusterInit(self):
     self.actName = ''
 
   def GetMetricName(self):
-    metricDict = {DataStructs.DiceSimilarity: 'Dice', DataStructs.TanimotoSimilarity: 'Tanimoto', 
-                  DataStructs.CosineSimilarity: 'Cosine', } # DataStructs.TverskySimilarity: 'Tversky'
-    try:
-      return metricDict[self.metric]
-    except KeyError:
-      if self.metric:
-        return self.metric
-      return 'Unknown'
+    # DataStructs.TverskySimilarity: 'Tversky'
+    metricDict = {DataStructs.DiceSimilarity: 'Dice', 
+                  DataStructs.TanimotoSimilarity: 'Tanimoto', 
+                  DataStructs.CosineSimilarity: 'Cosine', } 
+    self.metric = metricDict.get(self.metric, self.metric)
+    if self.metric:
+      return self.metric
+    return 'Unknown'
```

Comment:
My suggestion was incorrect as there is no assignment to `self.metric`.
```
  def GetMetricName(self):
    # DataStructs.TverskySimilarity: 'Tversky'
    metricDict = {DataStructs.DiceSimilarity: 'Dice', 
                  DataStructs.TanimotoSimilarity: 'Tanimoto', 
                  DataStructs.CosineSimilarity: 'Cosine', } 
    name = metricDict.get(self.metric, self.metric)
    if name:
      return name
    return 'Unknown'
```

Reply:
OK, I got your idea! I would fix it in the next patch. Thank you for your review.